thrift/src/main/java/com/linecorp/armeria/server/thrift/ThriftDocServicePlugin.java

```diff
@@ -168,7 +224,8 @@ static ServiceInfo newServiceInfo(
             });
         }
 
-        return new ServiceInfo(name, functions, classes, endpoints, docStrings.get(name), exampleHeaders);
+        return new ServiceInfo(name, functions, classes, endpoints,
```

Comment:
Do you think we can move the attachment of examples to services / methods to the docservice? Since we give them a name, and we have a name in the Info classes, the docservice could do the matching itself.

It's a little less flexible, but I can't think of a good use case that wouldn't be satisfied.

When adding support for example http headers to the grpc plugin in my current code, I felt silly doing basically the exact same logic there as here.